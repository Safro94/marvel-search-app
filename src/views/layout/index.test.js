import React from 'react';
import { render, wait } from '@testing-library/react';

import Layout from './';

jest.mock('../../routes', () => 'routes');
jest.mock('../../components/header', () => 'header');
jest.mock('../../components/shared/container', () => 'container');

jest.mock('./style', () => {
  return {
    Wrapper: 'wrapper',
  };
});

describe('<Layout /> ', () => {
  beforeEach(() => jest.clearAllMocks());

  it('It should render OK', async () => {
    // Act
    const { container } = render(<Layout />);
    await wait();

    // Assert
    expect(container).toMatchSnapshot();
  });
});
