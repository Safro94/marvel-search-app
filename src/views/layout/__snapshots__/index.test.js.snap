// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout />  It should render OK 1`] = `
<div>
  <container>
    <header />
    <wrapper>
      <routes />
    </wrapper>
  </container>
</div>
`;
